scripts/zephyr_module.py

```diff
@@ -225,27 +226,60 @@ def process_settings(module, meta):
     return out_text
 
 
-def kconfig_snippet(meta, path, kconfig_file=None):
+def get_blob_status(path, sha256):
+    if not path.is_file():
+        return 'D'
+    with path.open('rb') as f:
+        m = hashlib.sha256()
+        m.update(f.read())
+        if sha256.lower() == m.hexdigest():
+            return 'A'
```

Comment:
hey would it make sense to have a few constants for the various 'A' 'M' 'D' statuses...? Can't quite figure what 'A' means here :-) suppose the other two are modified and deleted but yeah maybe an internal explicit constant would help.

Reply:
Sure, I will add this as part of this change @fabiobaltieri